migrate/sqlschema/inspector.go

```diff
@@ -57,12 +68,23 @@ func WithSchemaName(schemaName string) InspectorOption {
 	}
 }
 
-// WithExcludeTables forces inspector to exclude tables
-// from the reported schema state.
+// WithExcludeTables forces inspector to exclude tables from the reported schema state.
 // It works in append-only mode, i.e. tables cannot be re-included.
+//
+// All tables are additionally appended to WithExcludeTablesLike
+// for forward-compatibility, allowing dialects to ignore deprecated ExcludeTables.
 func WithExcludeTables(tables ...string) InspectorOption {
 	return func(cfg *InspectorConfig) {
 		cfg.ExcludeTables = append(cfg.ExcludeTables, tables...)
+		cfg.ExcludeTablesLike = append(cfg.ExcludeTablesLike, tables...)
```

Comment:
Because patterns without wildcards are valid, we also deprecate `sqlschema.InspectorConfig.ExcludeTables` and encourage dialects (pgdialect for now) to use ExcludeTablesLike only.
 
To make this possible `sqlschema.WithExcludeTables` appends to both ExcludeTables and ExcludeTablesLike list.

Reply:
Can we directly reuse this API while allowing users to pass both actual table names and regex-based table names?
That way, we only need to modify the underlying implementation and can continue using WithExcludeTables.

Reply:
Yes, that's a great suggestion!